pkg/reconcilermanager/controllers/reconciler_base.go

```diff
@@ -172,6 +172,15 @@ func (r *reconcilerBase) createOrPatchDeployment(ctx context.Context, declared *
 
 	// TODO: check if VPA is enabled
```

Comment:
Remove the TODO comment since this commit resolves it.

Reply:
replaced todo with clarifying comments